src/test/java/org/jenkinsci/plugins/workflow/job/properties/PipelineTriggersJobPropertyTest.java

```diff
@@ -93,10 +94,10 @@ public void submitCallsStartTrue() throws Exception {
         t = (MockTrigger) p.getTriggers().get(t.getDescriptor());
         assertNotNull(t);
         // The  first "null, false" is due to the p.addTrigger(t) call and won't apply in the real world.
-        assertEquals("[null, false, null, true]", MockTrigger.startsAndStops.toString());
+        assertEquals("[null, false, null, true, null, false]", MockTrigger.startsAndStops.toString());
```

Comment:
So it seems that we are now calling `stop()` and `start(false)` gratuitously after a form resubmission.

Reply:
No one actually understands why or when we're calling those, honestly.